conda-forge.yml

```diff
@@ -1,13 +1,17 @@
+azure:
+  build_id: 48
+  user_or_org: TileDB-Inc
+  project_name: CI
 build_platform:
   osx_arm64: osx_64
 conda_forge_output_validation: true
 github:
   branch_name: main
   tooling_branch_name: main
 provider:
-  linux_aarch64: default
+  linux_aarch64: azure
```

Comment:
This fixes the linux-aarch64 builds. The "default" option is Travis, which isn't enabled for this repository